apis/v1alpha1/generator.yaml

```diff
@@ -183,6 +184,31 @@ resources:
         from:
           operation: RestoreDBInstanceFromDBSnapshot
           path: UseDefaultProcessorFeatures
+      # Used by create db instance read replica
+      SourceDBInstanceIdentifier:
+        from:
+          operation: CreateDBInstanceReadReplica
+          path: SourceDBInstanceIdentifier
+      DestinationRegion:
+        from:
+          operation: CreateDBInstanceReadReplica
+          path: DestinationRegion
+      ReplicaMode:
+        from:
+          operation: CreateDBInstanceReadReplica
+          path: ReplicaMode
+      SourceRegion:
+        from:
+          operation: CreateDBInstanceReadReplica
+          path: SourceRegion
+      PreSignedUrl:
```

Comment:
Please capitalise to `URL`

Reply:
looks like RDS api take it as `Url` instead of `URL` e.g. https://docs.aws.amazon.com/cli/latest/reference/rds/create-db-instance-read-replica.html

Reply:
I understand that, but we transform all of our Spec and Status field initialisms into full upper case. For example `ARN` is always upper-case. Our code generator knows to transform the casing when creating AWS SDK types

Reply:
i see, will use capital case here